test/CodeGenILTests/MicroTests.cs

```diff
@@ -199,6 +199,39 @@ public void SumIntRange_TypeCoerNeeded()
             Assert.AreEqual(55, output.Values.ToList()[1][0]);
         }
 
+        [Test, Category("Failure")]
+        //'Object of type 'System.int64[]' can be converted to type 'System.Collections.Generic.IEnumerable`1[System.Double]'.'
+        public void SumIntArray_TypeCoerNeeded()
+        {
+            var dscode = @"
+import(""DesignScriptBuiltin.dll"");
+import(""DSCoreNodes.dll"");
+y = DSCore.Math.Sum([1,2,3]);";
```

Comment:
does this possibly also fail because `[1,2,3]` is not an identifier?

Reply:
This was the case we discussed. I still need to fix it possible with a solution that declares a temporary local and using that local variables index, something like:
```
var localBuilder = DeclareLocal(<type>);
var trackedIndex = localBuilder.LocalIndex;
...
EmitOpCode(OpCodes.Ldloc, trackedIndex);
```

Reply:
theres a comment above describing a different reason for failure, is that accurate?

Reply:
That just says that the conversion is supported but it's failing due to a different reason.